homeassistant/components/deconz/config_flow.py

```diff
@@ -169,3 +169,25 @@ def __init__(self):
         user_input = {CONF_ALLOW_CLIP_SENSOR: True,
                       CONF_ALLOW_DECONZ_GROUPS: True}
         return await self.async_step_options(user_input=user_input)
+
+    @staticmethod
+    def async_get_options_flow():
+        """"""
+        return DeconzOptionsFlowHandler
+
+
+from homeassistant import data_entry_flow
+
+
+class DeconzOptionsFlowHandler(data_entry_flow.FlowHandler):
+    """Handle a deCONZ options flow."""
+
+    VERSION = 1
```

Comment:
I think that we don't have to version it separately, since we store it inside the config entry. If we need to migrate options, we should bump the version of the whole config entry.

Reply:
I'll remove it